changeset.go

```diff
@@ -1559,43 +1557,6 @@ func (cs *Changeset) upsertSecret(ctx context.Context, tr *ChangesetResource, da
 func (cs *Changeset) Init(ctx context.Context) error {
 	log.Debug("changeset init")
 
-	version, err := cs.Client.ServerVersion()
-	if err != nil {
-		return trace.Wrap(err)
-	}
-	versionMajor, err := strconv.Atoi(version.Major)
-	if err != nil {
-		return trace.Wrap(err)
-	}
-	versionMinor, err := strconv.Atoi(version.Minor)
-	if err != nil {
-		return trace.Wrap(err)
-	}
-
-	// TODO(knisbet) deprecated. Remove Third Party resources when we're no longer supporting kubernetes < 1.8
-	if versionMajor == 1 && versionMinor < 8 {
```

Comment:
Removing all this means that if 4.x will need a fix in rigging, we'll have to branch out from one of older tags. Maybe keep all this for the meanwhile? Technically we still have (and will have, for foreseeable future) 1.7 running in production.

Reply:
I think the third party resources was removed from the library... at least I can't seem to find it. Although I haven't tested it, Custom Resources was considered stable in 1.7.